index.html

```diff
@@ -5,30 +5,28 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
     <title>Product Landing Page</title>
 </head>
 
 <body>
     <header id="header">
         <!--headers-->
         <h1>Samsung Televisions</h1>
-
-        <img src="IMAGES/LOGO.jfif" alt="Samsung Logo" id="header-img">
+        <img src="./IMAGES/LOGO.jfif" alt="Samsung Logo" id="header-img">
         <nav id="nav-bar">
             <a class="nav-link" href="#home">Home</a>
             <a class="nav-link" href="#service">Service</a>
             <a class="nav-link" href="#product">Product</a>
```

Comment:
This is an example inline comment. :slightly_smiling_face:

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.